src/ValueObject/EncryptedColumn.php

```diff
@@ -43,4 +63,25 @@ public function getData()
     {
         return $this->data;
     }
+
+    /**
+     * @return string
+     */
+    public function getEncryptor(): string
```

Comment:
Difficult to enable compatibility with other service containers if using a VO for the identifier; thou it would add value - I flipped these a couple of times while writing the code. (The string also allowed using it as a default value in the VO constructor, preserving BC) Thoughts?